docs/Design/SoftArchitecture/MG.tex

```diff
@@ -278,7 +278,15 @@ \subsubsection{Input Format Module (\mref{mInput})}
   [Information to include for leaf modules in the decomposition by secrets tree.]
 \end{description}
 
-\subsubsection{Etc.}
+\subsubsection{Receipt Extraction Module (\mref{mExtraction})}
+
+\begin{description}
+\item[Secrets:] The algorithms required to extract relevant information from the scanned receipt text.
+\item[Services:] Takes the entire scanned receipt text and isolates only the necessary information required
```

Comment:
is the only thing it does "isolating"? isn't it supposed to organize it and send it somewhere